test/integration_test.dart

```diff
@@ -387,6 +394,14 @@ void main() {
                       'Conectarea în rețeaua *eduroam* se face pe baza aceluiași cont folosit și pe site-ul de cursuri.',
                   tags: ['Conectare', 'Informații'])
             ]));
+
+    mockNewsProvider = MockNewsProvider();
+    // ignore: invalid_use_of_protected_member
+    when(mockNewsProvider.hasListeners).thenReturn(false);
+    when(mockNewsProvider.fetchNewsFeedItems(context: anyNamed('context')))
+        .thenAnswer((realInvocation) => Future.value(<NewsFeedItem>[]));
```

Comment:
Add some test data here (it helps to check that we don't have overflows on different screen sizes). You can keep it empty in the other tests.